acceptance/tasks/acceptance.rake

```diff
@@ -0,0 +1,56 @@
+require 'rake'
+
+def generate_hosts(pe_version, tests)
+  hosts_file = 'hosts.cfg'
+  puts "Generating hosts..."
+  cmd = <<HEREDOC
+export BUNDLE_PATH=.bundle/gems
+export BUNDLE_BIN=.bundle/bin
+
+bundle install
+
+beaker-hostgenerator --pe_dir=http://enterprise.delivery.puppetlabs.net/#{pe_version}/ci-ready \
+--disable-default-role --hypervisor vmpooler centos7-64mdc-64a > #{hosts_file}
+HEREDOC
+
+  Dir.chdir('acceptance'){
+    system(cmd)
+  }
+
+  hosts_file
+end
+
+def run_beaker(hosts_file, tests, package_version)
+cmd = <<HEREDOC
+# We should change this to SHA and SUITE_VERSION in the acceptance tests
+export PE_RAZOR_SERVER_PACKAGE_BUILD_VERSION=#{package_version}
+
+bundle exec beaker --xml --debug --root-keys --repo-proxy --hosts #{hosts_file} \
+--type pe --keyfile ~/.ssh/id_rsa-acceptance --preserve-hosts onfail \
+--helper lib/helper.rb --pre-suite suites/pre_suite/install-server-from-module \
+--tests #{tests} --load-path lib
+HEREDOC
+
+  Dir.chdir('acceptance'){
+    system(cmd)
+  }
+end
+
+desc "Run acceptance tests"
+task :acceptance, [:version, :pe_version, :hosts_file] do |t, args|
+  version = args[:version] || '113eb26023ef96997b68324649bae3523b57b845'
+  pe_version = args[:pe_version] || '2018.2'
```

Comment:
Not sure how this PE version should be determined... maybe also required.

Reply:
Required in my opinion.